spec/schemas/_common.yaml

```diff
@@ -994,14 +994,14 @@ components:
       properties:
         count:
           description: |-
-            Total number of non-deleted documents across all primary shards assigned to selected nodes.
+            The total number of non-deleted documents across all primary shards assigned to selected nodes.
             This number is based on documents in Lucene segments and may include documents from nested fields.
           type: number
         deleted:
           description: |-
-            Total number of deleted documents across all primary shards assigned to selected nodes.
-            This number is based on documents in Lucene segments.
-            OpenSearch reclaims the disk space of deleted Lucene documents when a segment is merged.
+            The total number of deleted documents across all primary shards assigned to selected nodes.
+            This number is based on documents in Lucene segments. 
```

Comment:
It feels like there's text missing here. Do we mean something like "This number is based on the number of documents stored in Lucene segments"?